### PR TITLE
docs: added Issue and PR templates

### LINKED ISSUE #9: Add issue template

### LINKED ISSUE #10: Add PR template

### PR DESCRIPTION
**Pull request by ghost**
Resolves #9 
Resolves #10

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
```
Comment (shizhMSFT) [inline .github/PULL_REQUEST_TEMPLATE.md]: You may want to change "fix" to "resolve".

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+**Please check the following list**:
+- [ ]  Does the affected code have corresponding tests, e.g. unit test, E2E test?
```
Comment (shizhMSFT) [inline .github/PULL_REQUEST_TEMPLATE.md]: Only unit test is applicable

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- [ ]  Does this change require a documentation update?
```
Comment (shizhMSFT) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is also not applicable since tspclient-go does not have a separated doc site.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by JeyJeyGao — APPROVED** [review]
LGTM

**Pull request merged**